src/Cle.SemanticAnalysis/IR/Phi.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Immutable;
+using JetBrains.Annotations;
+
+namespace Cle.SemanticAnalysis.IR
+{
+    /// <summary>
+    /// A Phi node used in the SSA form IR.
+    /// </summary>
+    public class Phi
+    {
+        /// <summary>
+        /// Gets the destination local index.
+        /// </summary>
+        public int Destination { get; }
+
+        /// <summary>
+        /// Gets the operand local indices.
+        /// </summary>
+        [NotNull]
+        public ImmutableList<int> Operands { get; }
```

Comment:
This seems to be somewhat faster than `ImmutableList<ushort>`, even though it is a bit inconsistent.